include/libarmcortex/systick_timer.hpp

```diff
@@ -1,11 +1,11 @@
 #pragma once
 
 #include <cstdint>
-#include <functional>
 
 #include <libhal-util/bit.hpp>
 #include <libhal-util/static_callable.hpp>
 #include <libhal-util/units.hpp>
+#include <libhal/alias.hpp>
```

Comment:
Just a comment for later. We typically don't have to include something that is already included by the interface itself.